src/lib/utils/hooks/SynapseAPI/useEntityBundle.ts

```diff
@@ -21,14 +22,14 @@ const ALL_FIELDS: EntityBundleRequest = {
 }
 
 export default function useGetEntityBundle(
-  accessToken: string,
   entityId: string,
   bundleRequest: EntityBundleRequest = ALL_FIELDS,
   version?: number,
   options?: UseQueryOptions<EntityBundle, SynapseClientError, EntityBundle>,
 ) {
+  const { accessToken } = useSynapseContext()
   return useQuery<EntityBundle, SynapseClientError>(
-    ['entitybundle', entityId, version, bundleRequest],
+    [accessToken, 'entitybundle', entityId, version, bundleRequest],
     () =>
       SynapseClient.getEntityBundleV2(
         entityId,
```

Comment:
I think this is awesome: this query function is a hook itself, so we can call `useContext` here. The caller of `useGetEntityBundle` doesn't even need the access token!